source/assets/stylesheets/styles/_roles_tile.scss

```diff
@@ -86,6 +86,7 @@
   width: 100%;
   margin: auto;
   object-fit: cover;
```

Comment:
It's good we've got the description in the commit message but I also think a comment in the CSS file itself would be useful. A short one-liner should do it just explaining that the object fit JS library requires this rule to work.